Cargo.toml

```diff
@@ -7,8 +7,8 @@ edition = "2018"
 [dependencies]
 geo-types = "0.4"
 postgres = "0.17.4"
-mimirsbrunn = { git = "https://github.com/canalTP/mimirsbrunn", rev = "478ff0c" }
-mimir = { git = "https://github.com/canalTP/mimirsbrunn", rev = "478ff0c" }
+mimirsbrunn = { git = "https://github.com/canalTP/mimirsbrunn", rev = "e87d6db" }
+mimir = { git = "https://github.com/canalTP/mimirsbrunn", rev = "e87d6db" }
```

Comment:
`bragi` is also defined in "dev-dependencies" and should be installed with the same commit.

Reply:
Right, just updated it ;)